### PR TITLE
added the language strings for the new commands

### LINKED ISSUE #39: circShift, hShift and vShift command for tables
### DESCRIPTION
**What does your feature request improve on? Please describe.**
When having to rearrange table data the commands circShift, hShift and vShift  can be helpful. The same thing can be achieved with some lines of code, but that solution would be cleaner and might be faster.

**Describe the solution you'd like**
test() = {{1,2,3},{4,5,6},{7,8,9}}
circShift -> shift the table entries, there needs to be an option to shift row and col wise.
Example: circShift by 1 (row wise) -> {{9,7,8},{1,2,3},{4,5,6}} 
              circShift by 1 (col wise) -> {{9,1,2},{3,4,5},{6,7,8}} 
hShift -> hShift by 1 -> {{3,1,2},{6,4,5},{9,7,8}}
vShift -> vShift by 1 -> {{4,5,6},{7,8,9},{1,2,3}}

**Additional context**
It is possible to shift by an positive or negative integer.

(*Do not write below this line*)

-----------------------------------------------------
### ANALYSIS
Create two new `matop` functions `vectshift(MAT, n, m) -> MAT` (for rows and columns) and `circshift(MAT, n, m) -> MAT` (for circular shifts) to implement the required behaviors. All three functions are to be implemented by creating a new matrix with the elements reorganized, because the arguments of this functions are always passed as `const ref`. Both functions provide two dimensions to indicate the number of elements to be shifted row-wise and col-wise (in this order).

Consider also to creation of the necessary entries in the language files.

### IMPLEMENTATION STEPS
(*see also our [Wiki for implementation guidelines](https://github.com/numeredev/NumeRe/wiki/HowTo:-Contribute-Code)*)
- Implement the necessary changes in a new branch created here on GitHub
- Test your implementation

### DOCUMENTATION STEPS
(*see also our [Wiki for further information](https://github.com/numeredev/NumeRe/wiki/HowTo:-Create-Language-Strings)*)
- Update the changes log
- Add comments to your implementation
- Add Doxygen documentation comments
- Create or update the documentation articles (`*.NHLP` and `*.NDB` files, if needed)
- Update the language strings (`*.NLNG` files, if needed)

### PULL REQUEST
- Create a pull request for your changes
- Fill out the template
- Assign @numere-org/maintainers as reviewers

### PR DESCRIPTION
**Pull request by raphael-sys**
### SUMMARY
Implements necessary language changes.
Resolves numere-org/NumeRe#39

*Reviewers:* @numere-org/maintainers

### DOCUMENTATION
* **Documentation articles:**
    * [ ] corresponding documentation articles updated
    * [ ] new documentation articles created
    * [ ] not needed
* **Language files:**
    * [x] corresponding language files updated
    * [ ] not needed

### CODE REVIEW

**Review by numeredev — CHANGES_REQUESTED** [review]
Misses the entries within `syntaxelements.nlng`

File: de-DE/lang/numere.nlng [hunk 1]
```diff
@@ -82,6 +82,7 @@ PARSERFUNCS_LISTFUNC_FUNC_CAUCHY_PDF_[DISTRIB]= 			cauchy_pdf(x,a)
 PARSERFUNCS_LISTFUNC_FUNC_CAUCHY_RD_[RANDOM_DISTRIB]= 		cauchy_rd(a)                                                VAL      - Liefert eine Zufallszahl aus der Cauchy-Verteilung mit Skalenparameter a.
 PARSERFUNCS_LISTFUNC_FUNC_CHAR_[STRING]=					char(sStr,nPos)                                             STR      - Gibt das Zeichen aus der Zeichenkette sStr an der Stelle nPos zurück
 PARSERFUNCS_LISTFUNC_FUNC_CIRCLE_[DRAW]=					circle(P,r,STYLE)                                           {}       - Zeichnet einen Kreis um den Punkt P mit dem Radius r. Die Füllung und der Stil des Kreises kann mittels STYLE geändert werden. polygon() mit n >= 30 erzielt ähnliche Ergebnisse und erlaubt ein Kippen des "Kreises".
+PARSERFUNCS_LISTFUNC_FUNC_CIRCSHIFT[MAT]=					circshift(mat,n,dim)                                        MAT      - Verschiebt alle Elemente einer Matrix um n Positionen in Richtung der Matrixdimension dim (wählen Sie 0 für Zeile und 1 für Spalte).
```
Comment (numeredev) [inline de-DE/lang/numere.nlng]:
```suggestion
PARSERFUNCS_LISTFUNC_FUNC_CIRCSHIFT[MAT]=					circshift(mat,n,dim=0)                                      MAT      - Verschiebt alle Elemente einer Matrix um n Positionen in Richtung der Matrixdimension dim (wählen Sie 0 für Zeile und 1 für Spalte).
```

File: de-DE/lang/numere.nlng [hunk 2]
```diff
@@ -342,6 +343,7 @@ PARSERFUNCS_LISTFUNC_FUNC_TRIANGLE_[DRAW]=					triangle(P,P,P,STYLE)
...
 PARSERFUNCS_LISTFUNC_FUNC_VALTOSTR_[STRING]=				valtostr(x,C="",n=0)                                        STR      - Wertet x aus und wandelt den Wert in eine Zeichenkette um. C ist ein optionales Zeichen, das verwendet wird, um x auf n Zeichen, die vorne angestellt werden, zu füllen
 PARSERFUNCS_LISTFUNC_FUNC_VCAT_[MAT]=						vcat(mTop,mBot)                                             MAT      - Verkettet die zwei Matrizen vertikal. Es werden also die Zeilen der zweiten Matrix an die Zeilen der ersten Matrix angefügt. Diese Operation setzt voraus, dass beide Matrizen die gleiche Spaltenzahl haben, da ansonsten die Operation nicht durchführbar ist.
+PARSERFUNCS_LISTFUNC_FUNC_VECTSHIFT[MAT]=					vectshift(mat,n,dim)                                        MAT      - Verschiebt alle Spalten einer Matrix um n Positionen in Richtung der Matrixdimension dim (wählen Sie 0 für Zeile und 1 für Spalte).
```
Comment (numeredev) [inline de-DE/lang/numere.nlng]:
```suggestion
PARSERFUNCS_LISTFUNC_FUNC_VECTSHIFT[MAT]=					vectshift(mat,n,dim=0)                                      MAT      - Verschiebt alle Spalten einer Matrix um n Positionen in Richtung der Matrixdimension dim (wählen Sie 0 für Zeile und 1 für Spalte).
```

File: en-GB/lang/numere.nlng [hunk 3]
```diff
@@ -82,6 +82,7 @@ PARSERFUNCS_LISTFUNC_FUNC_CAUCHY_PDF_[DISTRIB]= 			cauchy_pdf(x,a)
 PARSERFUNCS_LISTFUNC_FUNC_CAUCHY_RD_[RANDOM_DISTRIB]= 		cauchy_rd(a)                                                VAL      - Returns a random variate from the Cauchy distribution with scale parameter a.
 PARSERFUNCS_LISTFUNC_FUNC_CHAR_[STRING]=					char(sStr,nPos)                                             STR      - Return the character at the position nPos in sStr
 PARSERFUNCS_LISTFUNC_FUNC_CIRCLE_[DRAW]=					circle(P,r,STYLE)                                           {}       - Draws a circle around the point P with the radius r. The filling and the style of the circle may be modified using STYLE. polygon() with n >= 30 generates similar results and allows "tilting" of the circle.
+PARSERFUNCS_LISTFUNC_FUNC_CIRCSHIFT[MAT]=					circshift(mat,n,dim)                                        MAT      - Shifts all elements of a matrix by n positions in the direction of the matrix dimension dim (choose 0 for row and 1 for column).
```
Comment (numeredev) [inline en-GB/lang/numere.nlng]:
```suggestion
PARSERFUNCS_LISTFUNC_FUNC_CIRCSHIFT[MAT]=					circshift(mat,n,dim=0)                                      MAT      - Shifts all elements of a matrix by n positions in the direction of the matrix dimension dim (choose 0 for row and 1 for column).
```

File: en-GB/lang/numere.nlng [hunk 4]
```diff
@@ -342,6 +343,7 @@ PARSERFUNCS_LISTFUNC_FUNC_TRIANGLE_[DRAW]=					triangle(P,P,P,STYLE)
...
 PARSERFUNCS_LISTFUNC_FUNC_VALTOSTR_[STRING]=				valtostr(x,C="",n=0)                                        STR      - Evaluates x and transforms it to a string. C is an optional character, inserted before x, to fill it to n characters
 PARSERFUNCS_LISTFUNC_FUNC_VCAT_[MAT]=						vcat(mTop,mBot)                                             MAT      - Concatenates the two matrices vertically. Thus, the rows of the second matrix are appended to the rows of the first matrix. This operation requires that both matrices have the same number of columns, otherwise the operation is not feasible.
+PARSERFUNCS_LISTFUNC_FUNC_VECTSHIFT[MAT]=					vectshift(mat,n,dim)                                        MAT      - Shifts all columns of a matrix by n positions in the direction of the matrix dimension dim (choose 0 for row and 1 for column).
```
Comment (numeredev) [inline en-GB/lang/numere.nlng]:
```suggestion
PARSERFUNCS_LISTFUNC_FUNC_VECTSHIFT[MAT]=					vectshift(mat,n,dim=0)                                      MAT      - Shifts all columns of a matrix by n positions in the direction of the matrix dimension dim (choose 0 for row and 1 for column).
```

**Comment by raphael-sys** [comment]
Added the vectshift to the syntaxelements. Should be everything you mentioned. For some reason there is still one requested change, but I cannot see what it is.

File: common/syntaxelements.nlng [hunk 5]
```diff
@@ -16,7 +16,7 @@ NPRC_COMMANDS=namespace str var tab cst procedure endprocedure if else elseif en
...
...reamslice streamtube strfind strings strjust strmatch strncmp strncmpi strread strrep strtok struct struct2cell strvcat sub2ind subplot subsasgn subsindex subspace subsref substruct subvolume sum superiorto support surf, surfc surf2patch surface surfl surfnorm svd svds symamd symbfact symmlq symmmd symrcm symvar system tan tanh tempdir tempname terminal tetramesh texlabel text textread textwrap tic, toc timer timerfind title toeplitz trace trapz treelayout treeplot tril trimesh triplequad triplot trisurf triu true tsearch tsearchn type uicontextmenu uicontrol uigetdir uigetfile uiimport uimenu uint8, uint16, uint32, uint64 uiputfile uiresume, uiwait uisetcolor uisetfont uistack undocheckout union unique unix unmkpp unregisterallevents unregisterevent unwrap unzip upper urlread urlwrite usejava vander var vectorize vectshift ver verctrl version vertcat view viewmtx volumebounds voronoi voronoin wait waitbar waitfor waitforbuttonpress warndlg warning waterfall wavplay wavread wavrecord wavwrite web weekday what whatsnew which whitebg who, whos wilkinson winopen wk1read wk1write workspace xlabel, ylabel, zlabel xlim, ylim, zlim xlsfinfo xlsread xmlread xmlwrite xor xslt zeros zip zoom
```
Comment (numeredev) [inline common/syntaxelements.nlng]:
```suggestion
MATLABFUNCTIONS=arrayfun plot strfind size num2str legend title bar grid axis repmat load fullfile length polyfit std atan hist mean zeros eye imread cos sin pi subplot colormap imagesc unique find linspace logspace meshgrid colorbar varargin varargout nargin nargout isempty isequal ginput ones min max nan numel cell2struct xlsread sum double transpose sscanf num2cell axes cat permute ceil imfilter cell sprintf regexp str2double ischar strmatch listdlg char str2num ismember abs strrep strcmp questdlg datestr copyfile fopen fprintf fclose isfield dir str2func sort ishandle round squeeze fminsearch xlim ylim isnan sqrt iscell text exist uigetdir uigetfile uisave save xlabel ylabel feof fgetl vertcat struct2cell fieldnames median strsplit regexprep now floor cell2mat testscan cellfun cross asin acos tan deal set get dlmread optimset reshape inputdlg textread strtok str2mat isdir mkdir fileparts dlmwrite fliplr interp2 griddata ndgrid imresize exp fft caxis lscov sign logical imopen diag bsxfun atan2 all any struct false true abs acos acosh acot acoth acsc acsch actxcontrol actxserver addframe addpath addproperty  airy alim all allchild alpha alphamap angle ans any area asec asech asin asinh assignin atan atan2 atanh audiodevinfo audioplayer audiorecorder auread auwrite avifile aviinfo aviread axes axis balance bar, barh bar3, bar3h base2dec beep besselh besseli besselj besselk bessely beta betainc betaln bicg bicgstab bin2dec bitand bitcmp bitget bitmax bitor bitset bitshift bitxor blanks blkdiag box brighten builtin bvp4c bvpget bvpinit bvpset bvpval calendar camdolly camlight camlookat camorbit campan campos camproj camroll camtarget camup camva camzoom capture cart2pol cart2sph cat caxis cd cdf2rdf cdfepoch cdfinfo cdfread cdfwrite ceil cell cell2mat cell2struct celldisp cellfun cellplot cellstr cgs char checkin checkout chol cholinc cholupdate circshift cla clabel class clipboard clock closereq cmopts colamd colmmd colorbar colordef colormap colormapeditor ColorSpec colperm comet comet3 compan compass complex computer cond condeig condest coneplot conj contour contour3 contourc contourf contourslice contrast conv conv2 convhull convhulln convn copyfile copyobj corrcoef cos cosh cot coth cov cplxpair cputime cross csc csch csvread csvwrite cumprod cumsum cumtrapz curl customverctrl cylinder daspect date datenum datestr datetick datevec dbclear dbcont dbdown dblquad dbmex dbquit dbstack dbstatus dbstep dbstop dbtype dbup dde23 ddeadv ddeexec ddeget ddeinit ddepoke ddereq ddeset ddeterm ddeunadv deal deblank dec2base dec2bin dec2hex deconv del2 delaunay delaunay3 delaunayn delete delete delete delete deleteproperty demo depdir depfun det detrend deval diag dialog diary diff dir disp disp disp display divergence dlmread dlmwrite dmperm doc docopt docroot dos dot double dragrect drawnow dsearch dsearchn echo edit eig eigs ellipj ellipke ellipsoid eomday eps erf, erfc, erfcx, erfinv, erfcinv error errorbar errordlg etime etree etreeplot eval evalc evalin eventlisteners events exist exit exp expint expm eye ezcontour ezcontourf ezmesh ezmeshc ezplot ezplot3 ezpolar ezsurf ezsurfc factor factorial false fclose fclose feather feof ferror feval fft fft2 fftn fftshift fgetl fgetl fgets fgets fieldnames figflag file formats fileattrib filebrowser fileparts filesep fill fill3 filter filter2 find findall findfigs findobj findstr finish fitsinfo fitsread fix flipdim fliplr flipud floor flops flow fmin fminbnd fmins fminsearch fopen fopen format fplot fprintf fprintf frame2im frameedit fread fread freeserial freqspace frewind fscanf fscanf fseek ftell full fullfile func2str function_handle functions funm fwrite fwrite fzero gallery gamma, gammainc, gammaln gca gcbf gcbo gcd gcf gco genpath get get get get getappdata getenv getfield getframe ginput gmres gplot gradient graymon grid griddata griddata3 griddatan gsvd gtext guidata guide guihandles hadamard hankel hdf hdfinfo hdfread hdftool help helpbrowser helpdesk helpdlg helpwin hess hex2dec hex2num hgload hgsave hidden hilb hist histc home horzcat hsv2rgb ifft ifft2 ifftn ifftshift im2frame im2java imag image imagesc imfinfo imformats import importdata imread imwrite ind2rgb ind2sub Inf inferiorto info inline inmem inpolygon input inputdlg inputname inspect instrcallback instrfind int2str int8, int16, int32, int64 interp1 interp2 interp3 interpft interpn interpstreamspeed intersect inv invhilb invoke ipermute is isa isappdata iscell iscellstr ischar isempty isequal isequalwithequalnans isevent isfield isfinite isglobal ishandle ishold isinf isjava iskeyword isletter islogical ismember ismethod isnan isnumeric isobject isocaps isocolors isonormals isosurface ispc isprime isprop isreal isruntime issorted isspace issparse isstr isstruct isstudent isunix isvalid isvalid isvarname javaArray javachk javaMethod javaObject keyboard kron lasterr lasterror lastwarn lcm legend legendre length length license light lightangle lighting lin2mu line LineSpec linspace listdlg load load load loadobj log log10 log2 logical loglog logm logspace lookfor lower ls lscov lsqnonneg lsqr lu luinc magic mat2cell mat2str material matlab matlabrc matlabroot max mean median memory menu mesh, meshc, meshz meshgrid methodsview mex mexext mfilename min minres mislocked mkdir mkpp mlock mod more move movefile movegui movie movie2avi moviein msgbox mu2lin multibandread multibandwrite munlock namelengthmax NaN nargchk nargoutchk nchoosek ndgrid ndims newplot nextpow2 nnls nnz noanimate nonzeros norm normest notebook now null num2cell num2str numel nzmax ode45, ode23, ode113, ode15s, ode23s, ode23t, ode23tb odefile odeget odeset ones open openfig opengl openvar optimget optimset orderfields orient orth pack pagedlg pagesetupdlg pareto partialpath pascal patch path path2rc pathtool pbaspect pcg pchip pcode pcolor pdepe pdeval peaks perl perms permute persistent pi pie pie3 pinv planerot plot plot3 plotedit plotmatrix plotyy pol2cart polar poly polyarea polyder polyeig polyfit polyint polyval polyvalm pow2 ppval prefdir primes print, printopt printdlg printpreview prod profile profreport propedit propedit psi pwd qmr qr qrdelete qrinsert qrupdate quad, quad8 quadl questdlg quit quiver quiver3 qz rand randn randperm rank rat, rats rbbox rcond readasync real reallog realmax realmin realpow realsqrt record rectangle rectint reducepatch reducevolume refresh regexp regexpi regexprep registerevent rehash release rem repmat reset reshape residue rethrow rgb2hsv rgbplot ribbon rmappdata rmdir rmfield rmpath root object roots rose rosser rot90 rotate rotate3d round rref rsf2csf run runtime save save save saveas saveobj scatter scatter3 schur script sec sech selectmoveresize semilogx, semilogy send sendmail serial serialbreak set set set set setappdata setdiff setfield setstr setxor shading shiftdim shrinkfaces sign sin single sinh size size slice smooth3 sort sortrows sound soundsc spalloc sparse spaugment spconvert spdiags speye spfun sph2cart sphere spinmap spline spones spparms sprand sprandn sprandsym sprank sprintf spy sqrt sqrtm squeeze sscanf stairs start startat startup std stem stem3 stop stopasync str2double str2func str2mat str2num strcat strcmp strcmpi stream2 stream3 streamline streamparticles streamribbon streamslice streamtube strfind strings strjust strmatch strncmp strncmpi strread strrep strtok struct struct2cell strvcat sub2ind subplot subsasgn subsindex subspace subsref substruct subvolume sum superiorto support surf, surfc surf2patch surface surfl surfnorm svd svds symamd symbfact symmlq symmmd symrcm symvar system tan tanh tempdir tempname terminal tetramesh texlabel text textread textwrap tic, toc timer timerfind title toeplitz trace trapz treelayout treeplot tril trimesh triplequad triplot trisurf triu true tsearch tsearchn type uicontextmenu uicontrol uigetdir uigetfile uiimport uimenu uint8, uint16, uint32, uint64 uiputfile uiresume, uiwait uisetcolor uisetfont uistack undocheckout union unique unix unmkpp unregisterallevents unregisterevent unwrap unzip upper urlread urlwrite usejava vander var vectorize ver verctrl version vertcat view viewmtx volumebounds voronoi voronoin wait waitbar waitfor waitforbuttonpress warndlg warning waterfall wavplay wavread wavrecord wavwrite web weekday what whatsnew which whitebg who, whos wilkinson winopen wk1read wk1write workspace xlabel, ylabel, zlabel xlim, ylim, zlim xlsfinfo xlsread xmlread xmlwrite xor xslt zeros zip zoom
```

**Review by numeredev — APPROVED** [review]

**Pull request merged**